stdlib/buffer.mli

```diff
@@ -158,6 +158,11 @@ val truncate : t -> int -> unit
   Raise [Invalid_argument] if [len < 0] or [len > length b].
   @since 4.05.0 *)
 
+val drop : t -> int -> unit
+(** [drop b len] drops the first [len] bytes of [b]
+  Raise [Invalid_argument] if [len < 0] or [len > length b].
```

Comment:
```suggestion
  @raise Invalid_argument if [len < 0] or [len > length b].
```
(I realise the one above is also incorrect, but at least we'll slowly converge towards modern ocamldoc comments!)